ci-operator/step-registry/redhat-appstudio/load-test/redhat-appstudio-load-test-commands.sh

```diff
@@ -89,13 +89,14 @@ echo "https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com" >"${GIT_CREDS_PATH}"
 
 cd "$(mktemp -d)"
 
-git clone --branch main "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
+git clone --origin upstream --branch main "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
 
 set -x
 if [ "$JOB_TYPE" == "presubmit" ] && [[ "$JOB_NAME" != rehearse-* ]]; then
     # if this is executed as PR check of github.com/redhat-appstudio/e2e-tests.git repo, switch to PR branch.
-    git fetch origin "pull/${PULL_NUMBER}/head"
+    git fetch upstream "pull/${PULL_NUMBER}/head"
     git checkout -b "pr-${PULL_NUMBER}" FETCH_HEAD
+    git pull --rebase upstream main
```

Comment:
Hi @jhutar could you PTAL at changes to load-test scripts?

Reply:
@jhutar thanks for review! I realized I need to add the `origin` remote again due to keeping the backward compatibility before [this PR](https://github.com/redhat-appstudio/e2e-tests/pull/1172/files#diff-293f6cf70e46616af76ba8cde40b57ce82d2cbbf7b632c10da79fd36294a933cL38) gets merged - would you mind giving it another look please? Thanks!

Reply:
Oh, nice!